server/src/internalClusterTest/java/org/opensearch/indices/IndicesRequestCacheIT.java

```diff
@@ -644,11 +647,14 @@ public void testProfileDisableCache() throws Exception {
                         .put(IndicesRequestCache.INDEX_CACHE_REQUEST_ENABLED_SETTING.getKey(), true)
                         .put(IndexMetadata.SETTING_NUMBER_OF_SHARDS, 1)
                         .put(IndexMetadata.SETTING_NUMBER_OF_REPLICAS, 0)
+                        // Disable index refreshing to avoid cache being invalidated mid-test
+                        .put(IndexSettings.INDEX_REFRESH_INTERVAL_SETTING.getKey(), TimeValue.timeValueMillis(-1))
                 )
                 .get()
         );
         indexRandom(true, client.prepareIndex(index).setSource("k", "hello"));
-        ensureSearchable(index);
```

Comment:
Same as above. Lets not remove this.

Reply:
discussed the order of ensureSearchable is important in slack, addressed that in the commit. - [9e3c4c1](https://github.com/opensearch-project/OpenSearch/pull/14369/commits/9e3c4c16a2424142011a1ecae44ed92c752c5c17)